src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -101,6 +109,7 @@ public KubernetesCloud(String name, List<? extends PodTemplate> templates, Strin
 
         this.serverUrl = serverUrl;
         this.namespace = namespace;
+
```

Comment:
note: i really wanted to just put the getNamespace() call here, but this constructor didn't seem to be getting called based on the behavior i was observing.  i'm not sure how that's possible, so if someone can point me to a better way to initialize namespace from the filesystem (when namespace is not explicitly set in the config) i'd be happy to do that instead.